classes/patreon_frontend.php

```diff
@@ -235,8 +220,10 @@ public static function getLabelUnderUniversalButton($patreon_level,$state =false
 		}
 		
 		$state['final_redirect_uri'] = $final_redirect;	
-
-		$refresh_link = '<a href="'.self::MakeUniversalFlowLink($patreon_level*100,$state).'">Refresh</a>';		
+		// 	$refresh_link = '<a href="'.self::MakeUniversalFlowLink($patreon_level*100,$state).'">Refresh</a>';		
```

Comment:
If we don't need this anymore, let's ditch it to keep the code clean.

Reply:
In a near opportunity we'll clean up various variables initialized in those functions which were used for MakeUniversalFlowLink only. They are left there as a comment for that purpose.  Lets keep them 1-2 weeks until that cleanup happens.